packages/flutter_tools/test/integration.shard/xcode_backend_test.dart

```diff
@@ -196,6 +197,8 @@ void main() {
 </dict>
 </plist>
 ''');
+
+        expect(result.stderr, isNot(startsWith("error:")));
```

Comment:
Without the fix, this would fail on macOS 26. I manually tested it since our CI doesn't run macOS 26 yet.